File: src/main/java/com/chatvizy/chatvisualizer/model/Message.java
package com.chatvizy.chatvisualizer.model;

/**
 * Holds parameters like data, time, userName and text.
 * */

public class Message {

    protected final String date;
    protected final String time;
    protected final String userName;
    protected final String text;

    public Message(String date, String time,
                   String userName, String text) {
        this.date = date;
        this.time = time;
        this.userName = userName;
        this.text = text;
    }

    public String getDate() {
        return date;
    }

    public String getTime() {
        return time;
    }

    public String getUserName() {
        return userName;
    }

    public String getText() {
        return text;
    }
}

File: src/main/resources/ChatVisualizer/src/app/model/Message.ts
export class Message {

  private _time: string;
  private _date: string;
  private _userName: string;
  private _text: string;

  constructor(time: string, date: string, userName: string, text: string) {
    this._time = time;
    this._date = date;
    this._userName = userName;
    this._text = text;
  }

  get time(): string {
    return this._time;
  }

  set time(value: string) {
    this._time = value;
  }

  get date(): string {
    return this._date;
  }

  set date(value: string) {
    this._date = value;
  }

  get userName(): string {
    return this._userName;
  }

  set userName(value: string) {
    this._userName = value;
  }

  get text(): string {
    return this._text;
  }

  set text(value: string) {
    this._text = value;
  }
}
